app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -276,7 +276,7 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
     }
 
     override fun onDestroyView() {
-        mediaSessionHolder?.videoVoiceCommandMediaSession?.onDestroyWebView(webView!!, session)
```

Comment:
Why would `webView` ever be null here? I'm concerned that, because we're using the webviewcache, the webView will persist outside of BrowserFragment with the javascript interface still attached.

Reply:
> I'm, not sure, but it seems that it very rarely can be.

Well, how often does this crash? :P

When possible, we need to make sure we know why a crash is happening because the assumptions that underlie a crash could be causing other problems too or the fixes can create other side effects in the code. I think switching out !! for ?. seems like an easy fix to avoid a crash but it ignores this understanding. It also may put users into an undefined state (and when these undefined states compound, it becomes harder to fix problems later).

> What if we did the JS removal in the cache instead of in WebRenderFragment?

That seems like a better fix: the interface is supposed to exist for the life of the WebView anyway.

Reply:
> Well, how often does this crash? :P

~3k / day.  Enough that I want to stop the crash, but few enough that I don't want to go too deep into the weeds.

> That seems like a better fix: the interface is supposed to exist for the life of the WebView anyway

Sounds good to me, I'll make the change.

Reply:
If we move removal into the `WebViewCache`, we should move the addition there too.  That's going to drag in a lot of surrounding context.  Getting a reference to `VideoVoiceCommandMediaSession`in `WebViewCache` looks like it should be doable, but I don't want to get into what the implications of that could be right now.  Unless you're opposed, I'm inclined to close this PR and push it back for now.